### PR TITLE
Reduce size of sliceContainers iterator

### PR DESCRIPTION
**Pull request by richardartoul**

### CODE REVIEW

File: roaring/containers.go [hunk 1]
```diff
@@ -319,23 +319,24 @@ func (sc *sliceContainers) Repair() {
...
-func (si *sliceIterator) Next() bool {
+func (si sliceIterator) Next() bool {
 	if si.e == nil || si.i > len(si.e.keys)-1 {
 		return false
 	}
-	si.key = si.e.keys[si.i]
-	si.value = si.e.containers[si.i]
-	si.i++
 
+	si.i++
```
Comment (xichen2020) [inline roaring/containers.go:342]: Not sure how this is intended to be used, but you probably need this to be a method on `*sliceIterator` for this to take effect?

Comment (richardartoul) [reply roaring/containers.go:342]: yeah I pushed the wrong version, should be good now

Comment (richardartoul) [reply roaring/containers.go:342]: how did you even find this p.r lol

Comment (xichen2020) [reply roaring/containers.go:342]: I auto subscribe to new repos =]

**Pull request closed without merging**